mockgcp/mocksecretmanager/secrets.go

```diff
@@ -62,22 +63,30 @@ func (s *SecretsV1) CreateSecret(ctx context.Context, req *pb.CreateSecretReques
 	obj.Name = fqn
 	obj.CreateTime = timestamppb.Now()
 	if obj.Replication == nil {
-		obj.Replication = &pb.Replication{}
-	}
-	if obj.Replication.Replication == nil {
-		obj.Replication.Replication = &pb.Replication_Automatic_{
-			Automatic: &pb.Replication_Automatic{},
-		}
+		return nil, fmt.Errorf("Secret.replication must be specified.")
```

Comment:
Really (I.e. this came up as a difference on record-gcp vs compare-mock)?  That's surprising!